Install headers prefixed, as to not cause a possible conflict with
different versions.

--- CMakeLists.txt.orig	2016-08-10 16:05:25 UTC
+++ CMakeLists.txt
@@ -96,7 +96,7 @@ set (LIB_SUFFIX "" CACHE STRING "Define
 set( LIB_INSTALL_DIR lib${LIB_SUFFIX} )
 set( PLUGIN_INSTALL_DIR lib${LIB_SUFFIX}/grantlee/${Grantlee_MAJOR_MINOR_VERSION_STRING} )
 set( BIN_INSTALL_DIR bin )
-set( INCLUDE_INSTALL_DIR include )
+set( INCLUDE_INSTALL_DIR include/grantlee4 )
 set( DATA_INSTALL_DIR share/apps )

 # set up RPATH/install_name_dir
